tests/unit/__init__.py

```diff
@@ -69,7 +70,10 @@ def search(self, drequest: DataRequest):
             # Read file and return json
             with gzip.open("tests/unit/imputations.json.gz", "rb") as f:
                 file_content = f.read()
-                return json.loads(file_content)
+                return [
+                    self._class_from_args(DataResult, item)
+                    for item in json.loads(file_content)
+                ]
```

Comment:
Odd that we're doing this in the __init__.py files. Fixture definitions should really be in their own file, or in conftest.py.

Reply:
Easy enough to move to conftest.py